### PR TITLE
Feat: Cart page Added

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added cart page to watchKart

-Added Buy now and address button
- Added logout feature 

Please review this PR

### CODE REVIEW

File: components/cart-page/cart.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <h1>Price summary</h1>
+        <div class="flex1"><p>Total MRP (incl of tax)</p><p>$2000</p>
+
+        </div>
+        <div class="flex1"><p>Delivery Fee</p><p>FREE</p></div>
+        <div class="flex1"><p>Discount</p><p>$200</p></div>
+        <div class="flex1"><p>subtotal</p><p>$1800</p></div>
+        <div class="flex2">
+            <button class="address-btn">Address</button>
+            <button class="buy-btn">Buy now</button>
```
Comment (sashaboi) [inline components/cart-page/cart.html:73]: Usually a CTA button like buy now needs to be the biggest and easiest to spot, maybe increase the size for it? 
Also address button isnt shown there usually , ok to remove. do refer the e com guidelines figma from lms.

File: components/cart-page/cart.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    <div class="cart-container">
+   
+        <div class="card-wrapper">
+            <img src="/asset/rolex.jpg" alt="rolex" class="card-img-hz">
+            <div class="card-info">
+                <h1 class="card-head-hz">Rolex Watch</h1>
+                <p class="margin-bottom-zero-five"><strong>Price: $1800 </strong>
+                     <span>(20% off)</span></p>
+                <label for="" class="qty">Quantity:  <input class="ip-qty" type="number" placeholder="2" name="qty" id=""></label></br>
```
Comment (sashaboi) [inline components/cart-page/cart.html:52]: In later versions, could try to add plus minus buttons and quantity number indicator styled differently than having it like a default input field.

**Review by sashaboi — APPROVED** [review]
The page looks good , I have made a few suggestions which might improve the experience, do try to go through them once you're done with everything
Onkar Deshpande

**Review by masraniJay — APPROVED** [review]
Looks good.

File: components/cart-page/cart.html (hunk 1 above)
Comment (anurag-kkumarr) [reply components/cart-page/cart.html:73]: Well did that

**Comment by anurag-kkumarr** [comment]
thanks for pr review

**Review by kr-sarvesh — COMMENTED** [review]
all ok to me

**Pull request merged**